x-pack/plugins/security_solution/public/common/components/event_details/event_details.test.tsx

```diff
@@ -20,47 +20,35 @@ jest.mock('../link_to');
 
 describe('EventDetails', () => {
   const mount = useMountAppended();
+  const onEventToggled = jest.fn();
+  const defaultProps = {
+    browserFields: mockBrowserFields,
+    columnHeaders: defaultHeaders,
+    data: mockDetailItemData,
+    id: mockDetailItemDataId,
+    view: 'table-view' as View,
+    onEventToggled,
+    onUpdateColumns: jest.fn(),
+    onViewSelected: jest.fn(),
+    timelineId: 'test',
+    toggleColumn: jest.fn(),
+  };
 
   describe('rendering', () => {
     test('should match snapshot', () => {
-      const wrapper = shallow(
-        <EventDetails
-          browserFields={mockBrowserFields}
-          columnHeaders={defaultHeaders}
-          data={mockDetailItemData}
-          id={mockDetailItemDataId}
-          view="table-view"
-          onEventToggled={jest.fn()}
-          onUpdateColumns={jest.fn()}
-          onViewSelected={jest.fn()}
-          timelineId="test"
-          toggleColumn={jest.fn()}
-        />
-      );
+      const wrapper = shallow(<EventDetails {...defaultProps} />);
       expect(wrapper).toMatchSnapshot();
     });
   });
+  const wrapper = mount(
```

Comment:
only needed one mount

Reply:
nit: Agreed, though it feels slightly out of place being declared down here, as opposed to being grouped with the other `const`s.

It looks like this placement avoids a linter conflict with the local declaration of `wrapper` in the `should match snapshot` test.

Consider giving the local `wrapper` in the `should match snapshot` test a different name (since that line/test is changing anyway), and grouping this declaration with the others.

Reply:
done!